### PR TITLE
S01 us12

### PR DESCRIPTION
**Pull request by Halicksse**
I add the footer.

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+import Footer from "./components/Footer";
 import "./App.css";
+// import SearchBar from "./components/SearchBar";
 
 function App() {
-  return <NavBar />;
+  return (
+    <div>
+      <NavBar />
+      {/* <SearchBar /> */}
```
Comment (ioayoub) [inline frontend/src/App.jsx:11]: Remove this please

File: frontend/src/components/SearchBar.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// import React from "react";
```
Comment (ioayoub) [inline frontend/src/components/SearchBar.jsx:1]: Dead code is not allowed here

File: frontend/src/main.jsx [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
 import App from "./App";
 
 const router = createBrowserRouter([
   {
-    path: "/",
     element: <App />,
+    children: [
+      {
+        path: "/",
+        element: "",
```
Comment (ioayoub) [inline frontend/src/main.jsx:12]: Elements are empty

**Review by ioayoub — CHANGES_REQUESTED** [review]
Please fix the commented lines + Conflicts

**Review by YOUNS28100 — APPROVED** [review]

**Review by Mewlyscin — APPROVED** [review]

**Review by ioayoub — APPROVED** [review]

**Pull request merged**